src/components/ShareResultsModal/EmailField.tsx

```diff
@@ -30,10 +31,16 @@ export default function EmailField({ onSubmit }: EmailFieldProps) {
 
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
```

Comment:
```suggestion
    setIsSubmitted(false);
    e.preventDefault();
```

Reply:
Currently 

1. If you click on "Send" for the first time you get the success notification
2. If you do not close the notification and click again on "Send" nothing happen (The notification message doesn't show again) (It does for the error message though)

I believe this is because "isSubmitted" doesn't get updated. The suggested line of code should fix the issue

Reply:
👍 Yep made that change.